### PR TITLE
Add MacLoggerDX version 6.36

### PR DESCRIPTION
**Pull request by macauley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/macloggerdx.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "macloggerdx" do
+  version "6.36"
+  sha256 "25ac048ed04a23c7b9a478d445847d7960e2146c67e83f7c8cd98ce770d3eb74"
+
+  url "https://www.dogparksoftware.com/files/MacLoggerDX#{version.delete(".")}.dmg"
```
Comment (neutric) [inline Casks/macloggerdx.rb:5]: `#{version.no_dots}` is available.

File: Casks/macloggerdx.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  name "MacLoggerDX"
+  desc "Ham Radio Logging and Rig Control Software"
+  homepage "https://www.dogparksoftware.com/MacLoggerDX.html"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
```
Comment (neutric) [inline Casks/macloggerdx.rb:12]: `strategy :page_match` should only be used when explicitly needed. It is the default except for special cases such as GitHub-URLs.

File: Casks/macloggerdx.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    regex(/Download: v(\d+(?:\.\d+)*)[^a-z]+/i)
+  end
+
+  app "MacLoggerDX.app"
+
+  zap trash: [
+    "~/Library/Preferences/com.dogparksoftware.MacLoggerDX*.plist",
+  ]
```
Comment (neutric) [inline Casks/macloggerdx.rb:20]: zap trash: "~/Library/Preferences/com.dogparksoftware.MacLoggerDX*.plist"

File: Casks/macloggerdx.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  url "https://www.dogparksoftware.com/files/MacLoggerDX#{version.delete(".")}.dmg"
+  name "MacLoggerDX"
+  desc "Ham Radio Logging and Rig Control Software"
+  homepage "https://www.dogparksoftware.com/MacLoggerDX.html"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
+    regex(/Download: v(\d+(?:\.\d+)*)[^a-z]+/i)
```
Comment (neutric) [inline Casks/macloggerdx.rb:13]:
```
regex(/Download:\s*v?(\d+(?:\.\d+)+)[^a-z]+/i)
``` 
is a bit more future-proof with regards to minimal changes.

File: Casks/macloggerdx.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
```
Comment (neutric) [inline Casks/macloggerdx.rb:15]: `depends_on macos: ">= :high_sierra"`

**Review by neutric — COMMENTED** [review]
Thanks again for your contribution to `homebrew-cask`, @macauley! My review is in the code. Take it with a grain of salt as I am not a maintainer.

**Comment by miccal** [comment]
Thank you @macauley.

**Pull request merged**